invenio_oauthclient/contrib/cern.py

```diff
@@ -159,6 +160,9 @@
 REMOTE_APP_RESOURCE_SCHEMA = "http://schemas.xmlsoap.org/claims/"
 
 
+oauth_cern_response_received = signal('oauth-cern-response-received')
```

Comment:
the signal  should be defined in a ``signals.py`` file.

Also it's better to define a generic signal ``oauth-reposnse-received`` and use the remote app as sender.